### PR TITLE
Common/section

### PR DESCRIPTION
**Pull request by RobertOsial**
Hi,
I've created reusable Section Component.
It will be used for Cast, Crew, Movies, etc

Thanks
Robert

### CODE REVIEW

File: src/common/Section/styled.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```
Comment (paulgrym) [inline src/common/Section/styled.js:3]: section tag would fit better

File: src/common/Section/styled.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+`;
+
+export const Title = styled.h2`
+    color: ${({ theme }) => theme.colors.black};
+    font-size: 36px;
+    margin: 0;
+`;
+
+export const Body = styled.div`
+    display: flex;
```
Comment (paulgrym) [inline src/common/Section/styled.js:16]: Maybe let's try with grid instead? :D

File: src/common/Section/styled.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+export const Wrapper = styled.div`
+    max-width: 1368px;
+    margin: 32px auto;
+    padding: 24px;
+`;
+
+export const Title = styled.h2`
+    color: ${({ theme }) => theme.colors.black};
+    font-size: 36px;
+    margin: 0;
```
Comment (paulgrym) [inline src/common/Section/styled.js:12]: I would add margin-bottom instead of padding in body :)

File: src/common/Section/styled.js (hunk 2 above)
Comment (RobertOsial) [reply src/common/Section/styled.js]: why not, changed to grid;D

File: src/common/Section/styled.js (hunk 3 above)
Comment (RobertOsial) [reply src/common/Section/styled.js:12]: make sense), added margin-bottom instead of padding

File: src/common/Section/styled.js (hunk 1 above)
Comment (RobertOsial) [reply src/common/Section/styled.js]: I guess I like "Wrapper" so much that I'm overusing it;D, changed to SectionTag:)

File: src/common/Section/styled.js [hunk 4]
```diff
@@ -1,19 +1,21 @@
 import styled from "styled-components";
 
-export const Wrapper = styled.div`
+export const SectionTag = styled.div`
```
Comment (paulgrym) [inline src/common/Section/styled.js:3]: I meant "styled.section" :D 
Sorry for the lack of clarity

**Pull request merged**